### ISSUE TITLE
Cannot read property 'is' of undefined

### ISSUE THREAD

**Issue by Tsyklop** [fancyapps/fancybox#1686]
I have net code:
```
$('a.select-tariff').fancybox($.extend({}, {
                autoFocus : false,
                backFocus : false,
                touch: false,
                closeClickOutside: true
   }, {
   afterLoad: function (instance, slide) {
      let content = $(slide.$content);
   }
 }));
```

It open popup in popup.

And i click on `a`, i get error:

```
jquery.fancybox.js:2933 Uncaught TypeError: Cannot read property 'is' of undefined
    at HTMLAnchorElement._run (jquery.fancybox.js:2933)
    at HTMLAnchorElement.dispatch (jquery.min.js:3)
    at HTMLAnchorElement.q.handle (jquery.min.js:3)
```
jQuery 3.2.1
fancyBox v3.1.28

**Comment by TimTulin** [comment]
same error with jQuery 1.last

**Comment by odahcam** [comment]
![image](https://user-images.githubusercontent.com/3942006/32392137-75e4301a-c0bb-11e7-899c-a7dca8c37edd.png)

| Lib          | Version |
| ---:          | :---       |
| jQuery     | 3.2.1    |
| fancyBox | 3.1.28  |

**Comment by fancyapps** [comment]
Hi,

This should be fixed in the latest version (v3.2.0).
